src/igl/ITrackedResource.h

```diff
@@ -33,14 +33,17 @@ class ITrackedResource {
    * @brief initResourceTracker() sets up tracking with the tracker
    * When the resource is destroyed, it will automatically deregister itself with the tracker
    */
-  void initResourceTracker(std::shared_ptr<IResourceTracker> tracker) {
+  void initResourceTracker(std::shared_ptr<IResourceTracker> tracker, const std::string & name = "") {
```

Comment:
Is it possible to use `const char*` here?

Reply:
@vinsentli  How do you plan to use this resource name? For logging?